README.md

```diff
@@ -19,23 +19,20 @@ The library is BSD 3-clause licensed.
   * Matrix: A lightweight, BSD-licensed matrix math library: https://github.com/px4/matrix - it is automatically included as submodule.
 
 
-By following the steps mentioned below you can create a shared library which can be included in projects using `-l` flag of gcc:
+By following the steps mentioned below you can create a static library which can be included in projects:
 
 ```
+make
+// OR
 mkdir build/
 cd build/
 cmake ..
 make
 ```
 
-Alternatively, just run:
-
-```
-./build.sh
-```
 ## Testing ECL
-By following the steps you can run the in `test/` specified checks
+By following the steps you can run the unit tests
```

Comment:
@kamilritz Are there tests results stored somewhere? If yes, would be cool to mention where.